magento/magento2
# Fix issue 19328 - Success Message Icon vertically misaligned in admin panel

### Description (*)

Success Message icon misaligned.
### Fixed Issues (if relevant)

1. magento/magento2#19328 : Success Message Icon vertically misaligned in admin panel

### Manual testing scenarios (*)

1.Login to admin panel > System > Import.
2.Select necessary Option > Upload CSV.
3.Click on Check Data Button > Check success Message Icon

### Contribution checklist (*)
